climate-monitoring-client/src/main/java/climatemonitoring/ProxyImpl.java

```diff
@@ -701,6 +775,42 @@ public synchronized Category[] getCategories() throws ConnectionLostException, D
 		return getcategories;
 	}
 	
+	/**
+	 * To get the category with the most recent recording about the specified area in the specified center
+	 * 
+	 * @param geoname_id The area's ID
+	 * @param center_id The center's ID
+	 * @return The latest category of the given center for the given area
+	 * @throws ConnectionLostException If the client loses connection during the operation
+	 * @throws DatabaseRequestException If the database fails to process the given request
+	 */
+	@Override
+	public Category getLatestCategory() throws ConnectionLostException, DatabaseRequestException {
+		
```

Comment:
You should add 2 parameters `int geoname_id` and `String center_id` like you wrote in the javadoc above. You should also send them to the server after the request type. If the functions gives you errors, you need to pull the new changes